recipe/meta.yaml

```
@@ -27,7 +26,7 @@ requirements:
     - {{ compiler('fortran') }}  # [not osx]
     - make
     - m2-patch  # [win]
-    - patch  # [not win] 
+    - patch  # [not win]
   host:
     - libgd {{ libgd }}
     - libxslt 1.1.*
```

```suggestion
    - libxslt 1.1.*
    - libxml2 {{ libxml2 }}
```
That is what the linter is complaining about. 
And libxml2 in run only causes it to be unpinned: https://staging.continuum.io/package/prefect/fs/nginx-feedstock/pr12/35d210b/linux-64/nginx-1.26.3-hfd100e4_0.tar.bz2